lms/views/helpers/__init__.py

```diff
@@ -1,6 +1,14 @@
 from lms.views.helpers._authentication import check_password
+from lms.views.helpers._authorization import is_instructor
 from lms.views.helpers._canvas_files import canvas_files_available
+from lms.views.helpers import frontend_app
```

Comment:
Note that I named this module without the prefixed underscore, as I would like to be able to import this module directly in views and invoke methods within it. I'm quite naive WRT python dependency management so could probably use some tutoring here.